pkg/network/node/ovscontroller.go

```diff
@@ -129,6 +129,7 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 	// eg, "table=10, priority=100, tun_src=${remote_node_ip}, actions=goto_table:30"
 	otx.AddFlow("table=10, priority=0, actions=drop")
 
+	otx.AddFlow("table=20, priority=300, udp, udp_dst=%d, actions=drop", vxlanPort)
 	// Table 20: from OpenShift container; validate IP/MAC, assign tenant-id; filled in by setupPodFlows
 	// eg, "table=20, priority=100, in_port=${ovs_port}, arp, nw_src=${ipaddr}, arp_sha=${macaddr}, actions=load:${tenant_id}->NXM_NX_REG0[], goto_table:21"
 	//     "table=20, priority=100, in_port=${ovs_port}, ip, nw_src=${ipaddr}, actions=load:${tenant_id}->NXM_NX_REG0[], goto_table:21"
```

Comment:
put the new AddFlow after the comment introducing table 20

Reply:
done